changelog

```diff
@@ -0,0 +1,3 @@
+version 1.0.0 @2018.10.29
```

Comment:
version of 1.0.0 means to stable, but the repo being developed, is it better to use 0.0.1 or 0.1.0? furthermore, i suggest we use [Semantic Versioning](https://semver.org/) to control our version-release, what's about you?

Reply:
the version of Go SDK is consistent with C SDK now.